lang/cs/Org.Apache.REEF.Network/StreamingCodec/IStreamingCodec.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Org.Apache.REEF.Wake.Remote;
+
+namespace Org.Apache.REEF.Network.StreamingCodec
```

Comment:
Is this codec shared cross Network project? Will we have other types of codec in the future? Shall we change the folder name to Codec in case we will need to add more types of Codec here to avoid more folders?